src/components/categoricaldisplay/Item.tsx

```diff
@@ -31,7 +31,7 @@ const ItemName = React.memo(({ item: { itemId, display } }: { item: TSubCategory
 const ItemDetail = React.memo(
     ({ favoritesId, item: { dataObject, itemId } }: { favoritesId: string; item: TSubCategoryDataItem }) => (
         <div className='sc-mcd__item__detail'>
-            {dataObject && (dataObject.exchange_is_open === undefined || dataObject.exchange_is_open) ? (
+            {dataObject && (dataObject.market === 'forex' || dataObject.exchange_is_open === undefined || dataObject.exchange_is_open) ? (
```

Comment:
```suggestion
            {dataObject && (dataObject.market === 'forex' || dataObject.exchange_is_open === undefined || dataObject.exchange_is_open) ? (
```
just a question. are we specifically checking for `undefined`?  looks kinda sus 👀

Reply:
Yes 😁 It was added a long time ago as well 😄

Reply:
I think the point of this was that it should only show `CLOSED` when the market is explicitly closed, and not when the response is undefined/not received yet